database/models/core.py

```diff
@@ -73,6 +73,7 @@ class Owner(CodecovBaseModel):
     trial_start_date = Column(types.DateTime, server_default=FetchedValue())
     trial_end_date = Column(types.DateTime, server_default=FetchedValue())
     trial_status = Column(types.Text, server_default=FetchedValue())
+    trial_fired_by = Column(types.Integer, server_default=FetchedValue())
```

Comment:
is this column already in the database?

Reply:
I guess this value is set from the API side when the trial is activated?

Reply:
yes yes